libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/common/SW360Utils.java

```diff
@@ -612,7 +612,6 @@ public static Integer parseStringToNumber(String input) {
         obligationAlreadyPresent.entrySet().stream().forEach(e -> obligationStatusMap.remove(e.getKey()));
 
         Map<String, ObligationStatusInfo> mapOfObligations = obligations.stream().filter(Objects::nonNull)
-                .filter(o -> isValidForProjectOptional || o.isValidForProject())
```

Comment:
Should also be removed from method signature.

Reply:
Removed it